public/removed.pug

```diff
@@ -0,0 +1,67 @@
+doctype html
+html(lang='en-US')
+  head
+    meta(charset='UTF-8')
+    meta(
+      content='initial-scale=1, shrink-to-fit=no, width=device-width',
+      name='viewport'
+    )
+    link(rel='preconnect', href='https://fonts.gstatic.com')
+    title #{ pageTitle }
+    meta(name='description', content=pageDescription)
+    meta(property='og:type', content='website')
+    meta(property='og:title', content=pageTitle)
+    meta(property='og:description', content=pageDescription)
+    meta(property='og:url', content=pageUrl)
+    meta(property='og:site_name', content=pageTitle)
+    meta(property='og:image', content=`${pageUrl}/images/og.png`)
+    link(rel='icon', type='image/x-icon', href=`${pageUrl}/images/favicon.ico`)
+    link(rel='icon', type='image/png', href=`${pageUrl}/images/favicon.png`)
+    link(rel='apple-touch-icon', href=`${pageUrl}/images/apple-touch-icon.png`)
+    link(rel='mask-icon', href=`${pageUrl}/images/logo.svg`, color='#111111')
+    link(
+      rel='stylesheet',
+      href='https://fonts.googleapis.com/css2?family=Open+Sans:wght@600&family=Roboto+Mono:wght@400;600&display=swap'
+    )
+    link(rel='license', href='license.txt')
+    link(rel='canonical', href=pageUrl)
```

Comment:
The head and the footer should be reused between the two pages. See [Template Inheritance with Pug](https://pugjs.org/language/inheritance.html).